### PR TITLE
Have tags automatically update search results

### LINKED ISSUE #49: Automatically search based on selected tags
Current: Click on tag then click search.
Update to: Click on tag then it automatically updates results.

### PR DESCRIPTION
**Pull request by ddehueck**
Fixes #49

### CODE REVIEW

File: frontend/src/components/datasets/DatasetForm.vue [hunk 1]
```diff
@@ -194,6 +194,7 @@ export default {
 			if (!this.validateData()) { return }
 			
 			// So users can't click multiple times
+			// Should change to loading icon instead of never allowing again
```
Comment (Eric4Jiang) [inline frontend/src/components/datasets/DatasetForm.vue]: Yes, I was thinking this too.

File: frontend/src/views/dataset/CreateDataset.vue [hunk 2]
```diff
@@ -88,7 +93,12 @@ export default {
 						console.log(response)
 					}
 				})
-			}) 
+			}).catch((response) => {
+				console.log("HERES ThE CATCH")
```
Comment (Eric4Jiang) [inline frontend/src/views/dataset/CreateDataset.vue]: "ThE" typo

Comment (ddehueck) [reply frontend/src/views/dataset/CreateDataset.vue]: Agh - debug typo - this catch shouldn't be in here.

File: frontend/src/components/datasets/DatasetForm.vue (hunk 1 above)
Comment (ddehueck) [reply frontend/src/components/datasets/DatasetForm.vue]: I'll create an issue

Comment (ddehueck) [reply frontend/src/components/datasets/DatasetForm.vue]: Created issue #53

File: frontend/src/views/dataset/CreateDataset.vue (hunk 2 above)
Comment (ddehueck) [reply frontend/src/views/dataset/CreateDataset.vue]: Added commit to remove this.

**Comment by ddehueck** [comment]
Now includes fix to #53 - Adds loading icon

**Comment by ddehueck** [comment]
I seem to have found a couple of issues in this PR - will update when fixed.

**Pull request merged**